lib/manageiq/active_record_connector.rb

```diff
@@ -0,0 +1,58 @@
+require "active_record"
+unless defined? DatabaseConfigurationPatch
```

Comment:
@Fryguy I was being defensive of the fact that this might not be required in a `require 'patches/database_configuration_patch'` for, but via `require_relative` or something like `require File.expand_path("../../patches/database_configuration_patch.rb", __FILE__)`.

That said, I am not entirely sure if ruby expands that under the covers so this isn't necessary or not... some investigation I guess is needed.  But that was the intent here.